src/app/products/share-products/view-share-product/view-share-product.component.html

```diff
@@ -3,7 +3,7 @@
     <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
     Edit
   </button>
-  <button mat-raised-button color="primary">
+  <button mat-raised-button color="primary" [routerLink]="['createdividend']">
```

Comment:
```suggestion
  <button mat-raised-button color="primary" [routerLink]="['create']">
```
fix routing, make this a child route of view dividends list page